### PR TITLE
Dropdown direction hint fixed false

### LINKED ISSUE #3719: Dropdown: Display above if not enough space below
image
Currently, I have problem with dropdown that if there is less space at bottom the dropdown selections is still shown at bottom which is quite hard for end-user selects the correctly option that he/she want. So it is really handy if the dropdown options show at the top of dropdown if there is less space at bottom.

Thanks,
Toan

### PR DESCRIPTION
**Pull request by micahgodbolt**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #3719 
- [X] Include a change request file using `$ npm run change`

#### Description of changes

I don't think this change will be quire as controversial as the ComboBox change, but it does change default behavior that some were expecting. Going to send out a full on email before we merge this in. Last one kinda blind sided a few.

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by micahgodbolt** [comment]
For reference, here is the explination of why we made this change:

---

One of the guiding principles of Fabric has always been "if there is a precedent on the web, default to that". This is why most booleans are written as 'isOpen' but disabled is just "disabled" and required is just "required". 
 
For that same reason, when we had users commenting about how broken dropdowns and comboboxes looked when they were at the bottom of the screen:
 

 
image

 
 
…we decided to change the default directionalHintFixed from true, to false, because that is how a standard HTML dropdown behaves.
 
But what we failed to do was make sure that everyone was aware of that change! And for that we apologize. Even though we felt that the existing behavior was in fact a bug, many relied on that behavior as “the way the combobox worked”.  One of those XKCD moments.
 
Fortunately, both Dropdown and ComboBox accept calloutProps, which will allow them to change the directionalHintFix back to “true”.
 
TL:DR
 
So for the sake of transparency, I wanted to make sure everyone knew that we currently have a PR out to change Dropdown’s directionalHintFixed to false. So if you are expecting that fixed behavior, you can go ahead and set those calloutProps now, and you won’t even notice this change.
